script/reverse_urls.py

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env python
+
+from argparse import ArgumentParser
+from urllib.parse import urlsplit
+import re
+
+def reverse_url(url):
+    components = urlsplit(url)
+    hostname = components.netloc.partition(':')[0]
+    result = '.'.join(hostname.split('.')[::-1])
+    if components.path:
+        result += components.path
```

Comment:
This can be simplified as: `result += components.path or ''`

Reply:
True, but since it is a cascade of multiple `if` statements, where the others cannot be changed, I prefer to keep them consistent.